Content.Shared/Body/Part/BodyPartCompatibility.cs

```diff
@@ -14,6 +14,7 @@ public enum BodyPartCompatibility
         Universal = 0,
         Biological,
         Mechanical,
-        Slime
+        Slime,
+        Skeleton
     }
```

Comment:
why do you need this?

Reply:
im not even sure why slimes were added to this tbh, thats not what this is for. these should just be biological